server.js

```diff
@@ -15,6 +15,17 @@ const server = httpServer({
     }
   },
 });
+function getFormattedTimestamp() {
```

Comment:
@arafatalim Why this time is formatted like this, why cant we use `new Date().toISOString()` to get the iso date format?

Reply:
removed the following function, now using `new Date().toISOString()`.